sdk/servicebus/azure-servicebus/azure/servicebus/management/_models.py

```diff
@@ -614,9 +614,9 @@ class SubscriptionProperties(DictMixin):  # pylint:disable=too-many-instance-att
     :ivar auto_delete_on_idle: ISO 8601 timeSpan idle interval after which the subscription is
      automatically deleted. The minimum duration is 5 minutes.
     :type auto_delete_on_idle: ~datetime.timedelta
-    :ivar entity_availability_status: Availability status of the entity. Possible values include:
+    :ivar availability_status: Availability status of the entity. Possible values include:
      "Available", "Limited", "Renaming", "Restoring", "Unknown".
-    :type entity_availability_status: str or
+    :type availability_status: str or
      ~azure.servicebus.management.EntityAvailabilityStatus
```

Comment:
shall we consider renaming `azure.servicebus.management.EntityAvailabilityStatus` to  `azure.servicebus.management.AvailabilityStatus` for consistency?

Reply:
I considered that, and was wondering if you or others would mention as well.

My reason for leaning towards no was that in isolation AvailabilityStatus seemed imprecise, and the added term narrowed its use such that, for being located in a somewhat generic module path, it'd be more clear what it's intended for.

Let me know what you think; I'll also cc @lmazuel to get a fresh pair of eyes.

Reply:
That's a fair point. `Entity` prefix makes it more clear whose status it's describing.
I see .NET Track1 is already doing such: https://docs.microsoft.com/en-us/dotnet/api/microsoft.servicebus.messaging.entityavailabilitystatus?view=azure-dotnet

just being over paranoid here: is the naming `EntityAvailabilityStatus` consistent among languages?

Reply:
Now I'm paranoid because I can't find it in the apiviews for either .net or java, but I have the issue here as well where it's noted that we're keeping it with the shortened name.  I've reached out to folks to try and see what the status is.

Reply:
So it's good we synced on this, there were some deltas.  Java didn't know they needed this because .net didn't have it in their apiview, as it was pending for them, so we're ahead of the pack.

Given this, I'm going to leave with my proposal for the time being and see if anyone flags it/we can have a proper design discussion where I can voice what I shared with you above.  More than happy to go either way if we do it intentionally.

LMK how this sounds to you.

Reply:
Personally I don't mind both, I let you decide.